CMakeLists.txt

```diff
@@ -192,14 +199,20 @@ endif()
 # We want Kokkos to be built with C++14, since that's what we're using in
 # Parthenon.
 set(Kokkos_CXX_STANDARD 14)
-if(DEFINED EXTERNAL_KOKKOS AND EXISTS ${EXTERNAL_KOKKOS})
-  find_package(Kokkos 3 REQUIRED PATHS "${EXTERNAL_KOKKOS}" NO_DEFAULT_PATH)
-elseif(EXISTS ${Kokkos_ROOT}/CMakeLists.txt)
-  add_subdirectory(${Kokkos_ROOT} Kokkos)
-elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/external/Kokkos/CMakeLists.txt)
-  add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/external/Kokkos Kokkos)
+if (PARTHENON_EXTERNAL_KOKKOS)
+    find_package(Kokkos 3)
+    if (NOT Kokkos_FOUND)
+        unset(PARTHENON_EXTERNAL_KOKKOS CACHE)
```

Comment:
Why the `unset()` if it is followed by a `FATAL_ERROR`.

Reply:
My understanding from @JoshuaSBrown is this makes it less likely to need to wipe the build directory clean between `cmake` calls

Reply:
I think that was a recommendation by @AndrewGaspar.

Reply:
Oops sorry both

Reply:
Yeah, that's right.